File: Java/combustivel.java
import java.util.Scanner;

public class combustivel {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int cod, alcool, gasolina, diesel;

		alcool = 0;
		gasolina = 0;
		diesel = 0;

		System.out.print("Informe um codigo (1, 2, 3) ou 4 para parar: ");
		cod = sc.nextInt();

		while (cod != 4) {
			if (cod == 1) {
				alcool = alcool + 1;
			} else if (cod == 2) {
				gasolina = gasolina + 1;
			} else if (cod == 3) {
				diesel = diesel + 1;
			}

			System.out.print("Informe um codigo (1, 2, 3) ou 4 para parar: ");
			cod = sc.nextInt();
		}

		System.out.println("MUITO OBRIGADO");
		System.out.println("Alcool: " + alcool);
		System.out.println("Gasolina: " + gasolina);
		System.out.println("Diesel: " + diesel);

		sc.close();
	}

}

File: Python/combustivel.py
a: int; g: int; d: int
cod = int(input("Informe um codigo (1, 2, 3) ou 4 para parar: "))
a = 0
g = 0
d = 0

while cod != 4:
    if cod == 1:
        a = a + 1
    elif cod == 2:
        g = g + 1
    elif cod == 3:
        d = d + 1

    cod = int(input("Informe um codigo (1, 2, 3) ou 4 para parar: "))

print("MUITO OBRIGADO")
print(f"Alcool {a}")
print(f"Gasolina {g}")
print(f"Diesel {d}")

